### PR TITLE
 AnchorPointFrame animation performance diffrent 

### PR DESCRIPTION
**Pull request by geron-cn**
fix bug: #23532 AnchorPointFrame animate performs diffrent with cocos studio

### CODE REVIEW

**Comment by geron-cn** [comment]
@VisualSJ  @pandamicro  @xiaofeng11  could you check this out

File: cocos/editor-support/cocostudio/ActionTimeline/CCActionTimelineCache.cpp [hunk 1]
```diff
@@ -969,17 +975,22 @@ ActionTimeline* ActionTimelineCache::createActionWithFlatBuffersForSimulator(con
...
-            action->addTimeline(timeline);
+        {
+            properTimelineMap.insert(std::make_pair(timelineFlatBuf->property()->c_str(), timeline));
+        }
+    }
+
+    for (const auto& properTimelinePair : properTimelineMap)
+    {
+        action->addTimeline(properTimelinePair.second);
     }
```
Comment (xiaofeng11) [inline cocos/editor-support/cocostudio/ActionTimeline/CCActionTimelineCache.cpp:992]: I think after modify, these function still do same work as before, except introduce a new local variable and use more operation.

Comment (geron-cn) [inline cocos/editor-support/cocostudio/ActionTimeline/CCActionTimelineCache.cpp:992]: `std::multimap` is a ordered container, sort the timelines depends property name.

Comment (xiaofeng11) [inline cocos/editor-support/cocostudio/ActionTimeline/CCActionTimelineCache.cpp:992]: OK

**Comment by xiaofeng11** [comment]
@pandamicro Reviewed, it can be merge.

**Pull request merged**